src/cmd/mapper/export/mapper-export.go

```diff
@@ -144,6 +145,18 @@ var ExportCmd = &cobra.Command{
 	},
 }
 
+func validateOutputFlags() error {
```

Comment:
the "validate" word makes it unclear that this function does something rather then checking the inputs.
consider changing to something like "handleOutputFlags"